paasta_tools/utils.py

```diff
@@ -2059,7 +2059,10 @@ def get_service_instance_list_no_cache(
             soa_dir=soa_dir,
         )
         for instance in instances:
-            instance_list.append((service, instance))
+            if instance.startswith('_'):
+                log.info("Ignoring %s.%s as instance name begins with '_'." % (service, instance))
```

Comment:
This will get annoying fast; it'll probably make most CLI tools print this on startup. Maybe do log.debug instead, or just drop the log line.

Reply:
I'll flip it to `log.debug` for now. I think it might prove helpful in debugging some potential issues when we start rolling this out and services start adopting it. We can kill it off later if necessary.